create-message/src/main/java/no/unit/nva/publication/messages/create/CreateMessageRequest.java

```diff
@@ -0,0 +1,25 @@
+package no.unit.nva.publication.messages.create;
+
+import no.unit.nva.identifiers.SortableIdentifier;
+
+public class CreateMessageRequest {
+
+    private String message;
+    private SortableIdentifier publicationIdentifier;
```

Comment:
publication has no URI id for the present

Reply:
We should have a quick chat about this